channels/es.m3u

```diff
@@ -441,6 +441,8 @@ http://amaru.dyndns.org:8835/0.m3u8
 https://5940924978228.streamlock.net/8161/8161/master.m3u8
 #EXTINF:-1 tvg-id="OlympicsChannel.es" tvg-name="Olympics Channel" tvg-country="AU" tvg-language="English" tvg-logo="http://i.mjh.nz/.images/tv.7olympics.png" group-title="",Olympics Channel [Geo-blocked]
 https://i.mjh.nz/au/Sydney/tv.7olympics.m3u8
```

Comment:
Doesn't work with AU vpn to me

Reply:
Ah yes that link... Apparently it went forbidden.

Reply:
I can try to make it work again, but i'll need to see from where the link starts.

Reply:
Look at this one https://ocshls-olympicchannel.akamaized.net/ocshls/OCAUS_1.m3u8

Reply:
Access denied

Reply:
Noted, i'll take it off for the time being then.